core-services/prow/02_config/konflux-ci/build-service/OWNERS

```diff
@@ -7,7 +7,9 @@
 approvers:
 - mmorhun
 - psturc
+- tisutisu
```

Comment:
@tisutisu  I think you need to add yourself to [build-service owners file](https://github.com/konflux-ci/build-service/blob/main/OWNERS) as well, otherwise the [prow's owners sync job](https://prow.ci.openshift.org/job-history/gs/test-platform-results/logs/periodic-prow-auto-owners) will remove you from this file eventually

Reply:
Thanks @psturc for the details. Was not aware of this sync job. Will send a PR to the build-service repo shortly.

Reply:
sent a [PR](https://github.com/konflux-ci/build-service/pull/345) to the build-service repo now.